src/store/user/sagas.js

```diff
@@ -0,0 +1,35 @@
+import { takeEvery, put, all } from 'redux-saga/effects';
+
+import { checkUser, login } from "../../services";
+
+import {
+    CHECK_USER,
+    LOGIN,
+    setUser
+} from "./actions";
+
+function* check() {
+    let user;
```

Comment:
make it as a `const` and move to `try` block

Reply:
@iliatcymbal This is generate [syntax error](https://prnt.sc/mo3zsr)